test/test_pr_tests.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 1,
```

Comment:
This notebook looks good to me. The only suggestion I could give here is to add workflow labels(such as `"labels":"PR-test"`) to those testing workflows by using the PATCH endpoint, to distinguish them from other workflows.

Reply:
Got it. I will probably remove the notebook before I merge, as the workflow will be executed with Jenkins. Because I have a few computers that I develop on, I tend to keep the testing code paired with the branches I work on. 

However, I think the repo is cleaner without the notebooks, so will usually remove them unless they contain analysis products (i.e. I'm working on a research ticket) . 

I've added a note to the PR; forgot this time. Sorry!

Reply:
i need to learn more about this python notebook stuff :(